AGPs/AGP-1.md

```diff
@@ -83,11 +82,16 @@ During Stage II you should seek feedback on your AGP idea by sharing it with you
 
 Be open-minded and respectful of all feedback you receive. Adjust your proposal to address serious concerns as they come up to increase the odds of your proposal passing review in later stages.
 
-### Stage III: Work In Progress
+### Stage III: Draft
 After you have asked the Aragon community whether an idea has any chance of support and have received sufficient feedback to feel confident going forward, you can create a draft AGP as a pull request to the AGP repo. Use a template from the Template section below to ensure you are including all the necessary information. The draft AGP file should be given a temporary name, which the AGP Editor will rename when the AGP is assigned a number.
 
-* If agreeable, an AGP Editor will assign the AGP a number (generally the PR number related to the AGP), move the proposal from Stage III to Stage IV by updating the status in the AGP, and merge your pull request. The AGP Editor will not unreasonably deny an AGP Stage IV status.
-* Reasons for denying Stage IV status include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing concerns by reviewers, or not in compliance with AGP-1.
+* If agreeable, an AGP Editor will assign the AGP a number (generally the PR number related to the AGP). The AGP Editor will not unreasonably deny assigning a number to the AGP.
+* Reasons for denying an AGP number and closing the pull request include being too unfocused, too broad, duplication of effort, being technically unsound, not providing proper motivation or addressing concerns by reviewers, or not in compliance with AGP-1.
+
+Once the draft pull request has been submitted to the AGP repo, you may submit follow-up commits with further changes to your draft until such point as you believe the AGP to be mature and ready to proceed to Stage IV. After a proposal in Stage III has been thoroughly reviewed, you may request that an AGP Editor moves the proposal from Stage III to Stage IV. If you do not request to move the proposal to Stage IV before Stage IV is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage IV begins.
```

Comment:
unsure if this is correct, but the current wording is definitely confusing: `move the proposal to Stage IV before Stage IV is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage IV begins`

```suggestion
Once the draft pull request has been submitted to the AGP repo, you may submit follow-up commits with further changes to your draft until such point as you believe the AGP to be mature and ready to proceed to Stage IV. After a proposal in Stage III has been thoroughly reviewed, you may request that an AGP Editor moves the proposal from Stage III to Stage IV. If you do not request to move the proposal to Stage IV before Stage V is scheduled to begin, AGP Editors will consider the proposal withdrawn and close the pull request when Stage V begins.
```

Reply:
@izqui It is correct. Stage III is the Draft stage, Stage IV is the Final Proposal stage when the Association review begins. If an AGP author does not request to have their proposal moved from the Draft stage (III) to Final Proposal stage (IV) before the Association review is scheduled to begin, then "AGP Editors will consider the proposal withdrawn and close the pull request".